llvm/test/CodeGen/NVPTX/param-align.ll

```diff
@@ -69,3 +69,39 @@ define ptx_device void @t6() {
   call void %fp(ptr byval(i8) null);
   ret void
 }
+
+; CHECK: .func check_ptr_align1(
+; CHECK: 	ld.param.u64 	%rd1
```

Comment:
I'm presuming there's still a split store to this ptr? Should we also CHECK for that so we can demonstrate that alignment of the pointer is being taken into account?

Reply:
There's no split store for this

Reply:
I've updated test file to show the store as well